model/PlanReview.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+/*  
+    This model refers to a plan reivew.
+*/
+const planReviewSchema = new Schema({
+  reviewee_id: { type: String, required: true, ref: "User" },
+  reviewer_id: { type: String, required: true, ref: "User" },
+  plan_id: { type: String, required: true, ref: "Plan" },
+  status: {
+    type: String,
+    required: true,
+    enum: ["PENDING", "ACCEPTED", "REJECTED"],
```

Comment:
Add "APPROVED"; accepted and approved are two different states

Reply:
Has this been resolved? @QinyuanWu @JJamesWWang

Reply:
If we go with what's currently here, advisors will not have an option to approve or disapprove of a plan. Up to you.

Reply:
This was resolved in Discord and will be made a task in our next Iteration.